src/lib/mina_wire_types/transaction_snark.ml

```diff
@@ -15,9 +15,9 @@ module type Concrete = sig
 
   module V2 : sig
     type t =
-      { statement : Mina_state.Snarked_ledger_state.With_sok.V2.t
-      ; proof : Proof.V2.t
-      }
+      ( Mina_state.Snarked_ledger_state.With_sok.V2.t
```

Comment:
I don't enjoy that we're changing the wire types here, especially when we're not intending to change the wire format.

Reply:
@mrmr1993 could you elaborate on your frustration?

Wire format didn't change, and we have some strong CI validation to assert that.

Reply:
Precomputed block serialization tests are rudimentary. Created #16501 to address it.

Going to alter this PR.